index.html

```diff
@@ -23,28 +39,28 @@ <h2>The Future</h2>
                 <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
-        </section>
+        </div>
         
-        <section class="middle-content">
+        <div class="middle-content">
         
             <h2>Why Did It Have To Be Boxes...</h2>
             
```

Comment:
Good work using multiple class names.  You can either do it this way or add an id ad well.